plugin/evm/imports_test.go

```diff
@@ -54,10 +56,12 @@ func TestMustNotImport(t *testing.T) {
 		// Importing these packages configures libevm globally and it is not
 		// possible to do so for both coreth and subnet-evm, where the client may
 		// wish to connect to multiple chains.
-		"plugin/evm/atomic": {"core", "core/vm", "params"},
-		"plugin/evm/client": {"core", "core/vm", "params"},
-		"plugin/evm/config": {"core", "core/vm", "params"},
-		"plugin/evm/header": {"core", "core/vm", "params"},
+		"plugin/evm/atomic": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/client": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/config": {"plugin/evm/customtypes", "params"},
+		"plugin/evm/header": {"core", "core/vm", "params"}, // Q: why do we need this?
```

Comment:
cc @qdm12, do you know why we need this?

I'm not sure if the header should be tested here as it's really the part of the custom code and should not be imported from outside.

Reply:
I don't 100% recall, but I would guess it's due to indirect imports. The header package is imported all over the place internally, so it's likely imported by other repositories

Reply:
I think this test should ensure we're not directly/indirectly causing libevm side-effects in customer-facing pkgs. I don't think `evm/header` is one of these pkgs. The test should be already checking the indirect imports as well, so I think we can remove the case here.